spec/requests/api/v1/listings/listing_limitation_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+RSpec.describe 'POST /api/v1/listings', type: :request do
+  let(:images) do
+    [
+      {
+        type: 'application/json',
+        encoder: 'iphone_picture',
+        data: 'AEwughvcvjdkshdhdcdcgWEgvcdhhd',
+        extension: 'jpg'
+      },
+      {
+        type: 'application/json',
+        encoder: 'parking_picture',
+        data: 'DfhgxhsxsvttfusxSDDDJhcdcdc',
+        extension: 'jpg'
+      }
+    ]
+  end
+
+  describe 'Reg user can not create a third listing' do
+    let(:landlord) { create(:user, role: 'registered') }
+    let!(:landlord_credentials) { landlord.create_new_auth_token }
+    let!(:landlord_headers) { { HTTP_ACCEPT: 'application/json' }.merge!(landlord_credentials) }
+
+    let!(:listing) { 2.times { create(:listing, landlord: landlord) } }
+
+    before do
+      post '/api/v1/listings',
+           params: {
+             listing: {
+               category: 'car park',
+               lead: 'Too good to park',
+               scene: 'outdoor',
+               address: 'Småbrukets backe 30 14158 Huddinge',
+               description: 'This is an outdoor parking but in safe area',
+               price: 150,
+               images: images
+             }
+           }, headers: landlord_headers
+    end
+
+    it 'is expected to return 422 response status' do
+      expect(response).to have_http_status 422
+    end
+
+    it 'is expect to return error message when creating third listing' do
+      expect(response_json['message']).to eq 'Subscribe to create more listing.'
```

Comment:
very minor but it should be "... more listingS."

Reply:
I admire your control of the english language Viktor.
I appreciate your linguistics skills.